display.php

```diff
@@ -0,0 +1,37 @@
+<?php 
+require_once 'functions.php';
+
+$db = new PDO('mysql:host=db; dbname=cuttlefish', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
Suggest moving these two lines into a function.

This will make it easier to get a database connection on any future pages you may add.

Reply:
Sorted